genie_registry/clinical.py

```diff
@@ -174,7 +174,27 @@ def _check_int_year_consistency(
 
     return ""
 
+def _check_year_death_validity(clinicaldf: pd.DataFrame) -> str:
+    """
+    Check if YEAR_DEATH >= YEAR_CONTACT.
+    When YEAR_DEATH is known, YEAR_CONTACT should = YEAR_DEATH.
 
+    Args:
+        clinicaldf: Clinical Data Frame
+        
+    Returns:
+        Error message if YEAR_DEATH if invalid
+    """    
+    error = ""
+    try:
```

Comment:
I would recommend not using `try, except` here because we don't want to lose other exceptions that could occur other than the intentional `assert` statement.

Reply:
okay will tweak this.